cmd/engine/main_oci_worker.go

```diff
@@ -285,6 +286,18 @@ func ociWorkerInitializer(c *cli.Context, common workerInitializerOpt) ([]worker
 
 	cfg := common.config.Workers.OCI
 
+	// Add a name for this engine to the labels that clients can use to log who they are connected to
+	engineName, ok := os.LookupEnv("_EXPERIMENTAL_DAGGER_ENGINE_NAME")
+	if !ok {
+		// use the hostname
+		hostname, err := os.Hostname()
```

Comment:
should we generate a random name is this fails a well as printing this in the engine logs?

Reply:
Yep updated with that. I was curious and it seems like the underlying syscall will only fail if the buffer provided to it to store the result is invalid, so hopefully we never hit that :-) But may as well handle it anyways, doesn't hurt